### ISSUE TITLE
Is there a way to import it directly to a page as a script?

### ISSUE THREAD

**Issue by redet-G** [ollama/ollama-js#106]
I have this old project that is not npm-based. I want to import the .js file as script and start using it. Is this possible?

**Comment by hopperelec** [comment]
No, because Ollama uses Typescript. However, you could download the source and compile it to Javascript then use that.
